### ISSUE TITLE
Python 3 Libraries Not Supported

### ISSUE THREAD

**Issue by doakey3** [codebrainz/geanypy#47]
Currently, when I attempt to import python 3 libraries, I get an error (even if I add the python 3 sitepackages folder to the sys.path, I still get incompatability errors.)

Currently my workaround is to use subprocess.check_output to call my python 3 scripts, but I'd like to distribute my plugins when subprocess is not needed. Is python 3 integration a goal for this project?

**Comment by elextr** [comment]
No, one of the dependencies will never be Python 3 IIUC so it won't be possible.

One of the Geany contributors has a replacement method of handling Python plugins, but he hasn't submitted it to geany-plugins yet.  @kugel- is it even public?

**Comment by kugel-** [comment]
yes it is, it even has seen multiple releases: https://github.com/kugel-/peasy

It's python 3, so give it a try.
